eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -226,15 +226,12 @@ stages:
                           PRLabels: "auto-merge"
                           CloseAfterOpenForTesting: '${{ parameters.TestPipeline }}'
 
-  - ${{if and(eq(variables['Build.Reason'], 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
-    - stage: SmokeTest_Release_Packages
-      displayName: Smoke Test Release Packages
-      jobs:
-        - template: /eng/pipelines/templates/jobs/smoke.tests.yml
-          parameters:
-            Daily: false
-            Artifacts: ${{ parameters.Artifacts }}
-            ArtifactName: ${{ parameters.ArtifactName }}
+          - ${{if and(eq(variables['Build.Reason'], 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
```

Comment:
FWIW, I changed ".com" to ".dev" and it made this change much easier to visualize: https://github.dev/Azure/azure-sdk-for-net/pull/23310

I see it puts it equivalent with the previous `- ${{if}}` on line 201, thus making it part of each artifact's Release stage.